application/views/qso/edit_ajax.php

```diff
@@ -358,7 +358,7 @@
                                             </select>
                                         </div>
                                         <div style="display: none;" class="mb-3 col-sm-6" id="location_us_county_edit">
-                                            <label for="stationCntyInput"><?= __("USA County"); ?></label>
+                                            <label for="stationCntyInput"><?= __("Station County"); ?></label>
```

Comment:
The field is shown but it seems I cannot set a value here. The field is emptied upon tabbing out ...